### ISSUE TITLE
Add a datetimepicker

### ISSUE THREAD

**Issue by JoseJohnen** [stride3d/stride#2223]
**Is your feature request related to a problem? Please describe.**
Please add a DateTimePicker, it is a very elegant solution to manage dates, for instance, the birth date of the player which you may want to save somewhere if you game is online for legal reasons and right now we don't have any solution to input dates in the system other than using one or more edittext and curate them by hand making certain manually than they are only numbers, the date is a valid one and make it us to work ourselfs the convertion of the data to a datetime in order to just then work it as it should.

The solution we have right now make a huge and insecure waste of time of the input of time, we need a component than can fix this.

**Describe the solution you'd like**
Create a DateTimePicker as they exist in some web context

**Describe alternatives you've considered**
Create other component for the GUI than can solve the issue of input time safetly and cleanly would be acceptable too, right now there are zero solutions to this respect so i would appreciate that for certain.

**Comment by IXLLEGACYIXL** [comment]
strude gui is pretty bare bones 

you can use myra or avalonia inside stride which have that feature allready
its like beating avalonia at its own game by trying to have everything they have too

**Comment by JoseJohnen** [comment]
> strude gui is pretty bare bones
> 
> you can use myra or avalonia ( maybe also wpf ? ) inside stride which have that feature allready its like beating avalonia at its own game by trying to have everything they have too

Agreed, but what's the point of having a stride GUI at all if it's not gonna have the bare mínimum at least. I mean, under that logic why have it (Stride GUI) instead of making a full implementation of myra or avalonia in Stride for instance.

If we are gonna have a barebones GUI, ok, but at least we may try to improve it over time in order to make it more complete and apt for the needs we have. Of course there is always the chance than we change it and start the implementation of avalonia or myra for this purpose in the future, but until then i think this should be the approach we should consider.

That being said i know and appreciate why you tell me this, is related to give me solutions to not being stuck, and as such i take the advice and when the time come i may implement it, i'm not there yet, so thank you for the advice.
